ihatemoney/static/css/main.css

```diff
@@ -536,6 +536,10 @@ tr:hover .extra-info {
 .navbar-dark .navbar-nav .show > .nav-link svg {
   fill: white;
 }
+.navbar-dark .dropdown-menu {
+  height: 50vh;
```

Comment:
```suggestion
  max-height: 60vh;
```
Use max-height to have a menu smaller if needed.
I also put 60vh, because on large screens, it's a bit confusing to have "only" `50vh`. But maybe `60vh` is too big on small screens?

Reply:
I was about to ask about this 50vh, because it can already be too big for very small screens...

Reply:
![Screenshot_2021-07-18 Account manager - demonstration](https://user-images.githubusercontent.com/297578/126082023-c2df7d73-bfd1-4b77-87f9-211231799c5d.png)
I don't think it's too big, but I'm not a real smartphone user :thinking:

Reply:
Right, 50vh is the maximum that fits on my phone.